packages/aragon-cli/src/commands/run.js

```diff
@@ -74,10 +74,26 @@ exports.builder = function(yargs) {
       description: 'Arguments to be passed to the template constructor',
       default: newDAO.BARE_TEMPLATE_DEPLOY_EVENT,
     })
+    .option('build', {
+      description:
+        'Whether publish should try to build the app before publishing, running the script specified in --build-script',
+      default: true,
+      boolean: true,
+    })
     .option('build-script', {
       description: 'The npm script that will be run when building the app',
       default: 'build',
     })
+    .option('prepublish', {
+      description:
+        'Whether publish should run prepublish script specified in --prepublish-script before publishing',
+      default: true,
+      boolean: true,
+    })
+    .option('prepublish-script', {
+      description: 'The npm script that will be run before publishing the app',
+      default: 'prepublish',
```

Comment:
npm's `prepublish` script is actually [kind of deprecated](https://github.com/npm/npm/issues/10074). It would be nice to also fallback to `prepublishOnly` if that exists.